WcaOnRails/app/javascript/image-preview/index.js

```diff
@@ -0,0 +1,15 @@
+import loadImage from 'blueimp-load-image';
+
+$(() => {
+  $('.image-preview-container').each((index, container) => {
+    const $container = $(container);
+    const $input = $($container.data('inputSelector'));
+    $input.on('change', () => {
+      loadImage(
+        $input[0].files[0],
+        img => img.type !== 'error' && $container.empty().append(img),
```

Comment:
It's noticeable if the image is not rendered, but logging surely makes sense, done!